lib/degreed/content/courses.rb

```diff
@@ -11,6 +11,10 @@ def initialize(token:)
         @token = token
       end
 
+      def dasherize(underscored_word)
```

Comment:
Nice simple, reusable solution 😎 

If we find we need this in more than one place, it might be worth either extracting this into a superclass for our "client" classes, or making it a "module_function" within a module containing utility helpers like this 🎉 

I don't think we need to right now; more just dreaming about the future.